### PR TITLE
Fix #2137 Added SIOP logo and reformatted the code.

### LINKED ISSUE #2137: Add SIOP logo in "Who uses EvalAI" list.
Here is the SIOP [logo](https://drive.google.com/file/d/1zcw3uqDyM1iHha-o1_zUzKSAkt__98sG/view)

### PR DESCRIPTION
**Pull request by yashbhutoria**

### CODE REVIEW

File: frontend/src/views/web/landing.html [hunk 1]
```diff
@@ -231,4 +256,4 @@ <h4>
     </div>
     <!-- footer -->
     <landing-footer></landing-footer>
-</div>
+</div>
```
Comment (RishabhJain2018) [inline frontend/src/views/web/landing.html]: Please add a new line.

**Comment by gautamjajoo** [comment]
@RishabhJain2018 This can be closed as @anuj-rathore was assigned this issue and he has opened a PR https://github.com/Cloud-CV/EvalAI/pull/2138 too

**Comment by RishabhJain2018** [comment]
Thanks @gautamjajoo for pointing it.

**Comment by yashbhutoria** [comment]
Fixed in #2185

**Pull request closed without merging**